extra_actions_base_proto/BUILD.bazel

```diff
@@ -7,14 +7,6 @@ go_proto_checkedin_test(
     src = "extra_actions_base.gen.pb.go",
 )
 
-genrule(
```

Comment:
Should we consider leaving a version of this that is not in the main codepath, but makes it easy to 're-vendor'?

Reply:
Do you have an example of what you mean? How do I remove io_bazel from the external dependencies but still make it easy to re-vendor?